Seeds.rb, Added users and models for departments

kgf opened the pull request:
to seeds.rb Not finished yet, but still helpful

kgf committed:
Added users and models for departments

to seeds.rb Not finished yet, but still helpful

kgf commented:
Anyone should feel free to review, but I know Chris was particularly interested.

houhoulis commented on db/seeds.rb:
```diff
@@ -5,3 +5,85 @@
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+# I have added some objects whose status is Inactive to ensure they don't appear.
+# Need to create
```
I think you created these things.  So I guess this comment is outdated?

houhoulis commented on db/seeds.rb:
```diff
@@ -5,3 +5,85 @@
+Role.find_or_create_by(name: "Admin")
+
+# Departments are where people call home.
+# I need another status to determine visbility on the various drop downs.
```
"visibility" (spelling)

But also, I don't understand this comment.  Is this a TODO?

houhoulis commented on db/seeds.rb:
```diff
@@ -5,3 +5,85 @@
+  ])
+
+# There does seem to be the correct link between skills and courses
+# As I create a person, I wonder what
```
Many of the comments in here seem like notes to yourself.  At least, I am not sure what they mean.  They could either be fleshed out, addressed, or removed.

houhoulis commented on db/seeds.rb:
```diff
@@ -5,3 +5,85 @@
+  {name: "Medical Reserve Corp", status: "Active"},
+  {name: "Department Public Works", status: "Active"},
+  {name: "Inactive Department", status: "Inactive"}
+  ])
```
Cool, I had no idea you could do

``` ruby
Foo.create([
  foo_attributes,
  foo_2_attributes,
  foo_3_attributes...
])
```

👍

kgf committed:
Made corrections as suggested by PR

Removed comments to a private location, in preparation of making issues
of them.
